src/Axiom/Set.agda

```diff
@@ -104,7 +111,8 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   strongly-finite : Set A → Type ℓ
   strongly-finite X = ∃[ l ] Unique l × ∀ {a} → a ∈ X ⇔ a ∈ˡ l
 
-  DecEq∧finite⇒strongly-finite : ⦃ _ : DecEq A ⦄ → (X : Set A) → finite X → strongly-finite X
+  DecEq∧finite⇒strongly-finite : ⦃ _ : DecEq A ⦄ (X : Set A) →
+    finite X → strongly-finite X
```

Comment:
Are we sure about this styling convention? (it always feels like an ill-formed expression to me, that's why I avoid it)

For non-Agda readers, it makes more sense to have
```
Forall x y z. 
  (1) x = y
  (2) z = x + y
```
i.e. putting the `.` in the first line

Reply:
Here are a few analogies that help us make peace with the convention.  I'm not sure they will be very helpful, but at least they help explain my own Agda style choices.

1. when summing up numbers, we usually prefer to write
   ```
     a
   + b
   + c
   ```
   instead of
   ```
   a +
   b +
   c
   ```
   (still, you may feel the latter seems more sensible)
2. Imagine there were many antecedents, say, `(X : Set A)` `(Y : Set B)` ... `(Z : Set C)`, preceding the arrow.  We usually don't want to put them all on one very long line, and we sometimes present them as
   ```
   (X : Set A)      -- comment explaining what X is for
   (Y : Set B)      -- comment explaining Y
   etc.
   ```
   Of course, this is equivalent to 
    ```
   (X : Set A)      -- comment explaining what X is for
   → (Y : Set B)    -- comment explaining Y
   → etc.
   ```
   as well as
   ```
   (X : Set A)        -- comment explaining what X is for
   ∧ (Y : Set B)      -- comment explaining Y
   ∧ etc.
   ```
   But again, I can see why you might want to advocate for expressing the latter as
   ```
   (X : Set A) ∧
   (Y : Set B) ∧
   etc.
   ```
3. My favorite Agda style is a combination of the styles of Martin Escardo and Phil Wadler, which is what I adopted in my `agda-algebras` library (though I often omit the horizontal line):
   ```
                  (X : Set A)
    →             (Y : Set B)
                 -------------
    →             (Z : Set C)
   ```
   The arrows are out of the way and seem to be second class citizens, so the whole expression now looks like "implication introduction" where the stuff above the horizontal line is a conjunction of a bunch of predicates (the antecedent), and below the horizontal line is the single consequent.  We can rearrange things depending on what we want to call attention to; if we're thinking of the above as a proof of the implication `Y → Z` from the premise `X`, then we might write it as
   ```
                  (X : Set A)
            -------------------------
    →       (Y : Set B) → (Z : Set C)
   ```
Of course these are matters of personal preference and I don't think it's a real problem if the style differs slightly across the code base.  Even the Agda standard library isn't entirely consistent style-wise.

Reply:
The main reason for this is consistency with everything else. Multi-line expressions are always nicer if you can align them at whatever operator joins them together & we have lots of places where we do it, so it feels odd to have this one exception.